CONTRIBUTING.md

```diff
@@ -39,3 +38,39 @@ npm version [TYPE]
 git push origin master --tags
 npm publish
 ```
+
+# Project Principles
+
+The following principles should help to inform decisions at every level of the
+project's architecture, from communication to implementation, and everything in
+between.
+
+## Inclusivity
+
+The driving principal behind the existence of React Accessible Accordion is an
+inclusive web. This most obviously relates to the inclusivity of those with
+disabilities or impairments which require the use of assistive technology, such
+as screenreaders. It should extend though to technological inclusivity, such
```

Comment:
This sentence I had to read a few times. I would say something more like:
This inclusivity should also extend to the technical community, so RAA  _tries_ to remain as technologically agnostic as possible.

Or something like that anyway?

Reply:
Hah, yes this is an awkward sentence huh??